index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About me</title>
+    <script src="https://kit.fontawesome.com/c98845bc10.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <header class="headline-bar">
+    <div class="logo">
+        <h3>Zeeyah</h3>
```

Comment:
- [ ] As suggested by the previous reviewer a logo on the page of a website is a link to the home page. Kindly wrap your logo inside anchor tag: `<a>` tag.